test/client/fixtures/sandbox/iframe-test.js

```diff
@@ -332,3 +333,53 @@ test("native methods of the iframe document aren't overridden for iframe with ja
     strictEqual(iframeSandbox._shouldSaveIframeNativeMethods(iframe), browserUtils.isWebKit);
 });
 
+test("'body.appendChild' method works incorrectly in the particular case (GH-421)", function () {
+    var iframes                       = [];
+    var countIframes                  = 2;
+    var createIframes                 = function () {
```

Comment:
This declaration looks ugly IMHO, wouldn't it be better if will be separated by new line with `function` keyword aligned close to equal sign?